### PR TITLE
Fix(bin/load-mocks): GitHub integration IntegrityError issue

### PR DESCRIPTION
**Pull request by lauryndbrown**
For users who had the legacy github plugin, this fixes the possible integrity error when running load mocks.

### CODE REVIEW

**Comment by lauryndbrown** [comment]
@mattrobenolt is there any reason we don't have tests for this script? Seems like those would be helpful honestly....

**Comment by mattrobenolt** [comment]
Because it's just mock data. And takes a super long time to run this script.

File: bin/load-mocks [hunk 1]
```diff
@@ -449,15 +449,27 @@ def main(num_events=1, extra_events=False):
...
+                # for users with legacy github plugin
+                # upgrade to the new integration
+                repo = Repository.objects.get(
+                    organization_id=org.id,
+                    provider='github',
+                    external_id='example/example',
+                    name='Example Repo',
+                )
+                repo.provider = 'integrations:github'
+                repo.save()
```
Comment (markstory) [inline bin/load-mocks]: The `repo` value is used as `repo[0].id` below. I don't think this variable matches the tuple that `get_or_create()` makes. You could unpack the results from `get_or_create()` and update the other uses of `repo`. That would let you use the bare model consistently.

Comment (lauryndbrown) [reply bin/load-mocks]: Wha.... why? oh my goodness why is it written this way. the other value isn't even used.... Thanks

**Review by mattrobenolt — APPROVED** [review]

**Pull request merged**